articles/overview/index.md

```diff
@@ -1,5 +1,10 @@
 # Welcome to Intent Architect
 
+>[!NOTE]
+>With Intent Architect we want you to stop focusing on the nitty-gritty aspects of Software development and really become more productive by completing new features that you need to build to really add value to your business.
+
+---
```

Comment:
We should discuss the adding of these horizontal lines, what are our rules going to be for when they should be present or not?

Reply:
I added them there for clear visual separators. But its not a train smash if we need to take them out